src/lib/viewers/doc/DocPreloader.js

```diff
@@ -243,6 +247,25 @@ class DocPreloader extends EventEmitter {
             });
     };
 
+    /**
+     * Resizes the preload and placeholder elements
+     *
+     * @return {void}
+     */
+    resize() {
+        if (!this.pdfData || !this.preloadEl) {
+            return;
+        }
+
+        const { pdfWidth, pdfHeight } = this.pdfData;
+        const { scaledWidth, scaledHeight } = this.getScaledDimensions(pdfWidth, pdfHeight);
```

Comment:
Can you move the above two lines to a util function? It's duplicated inside of the `loadHandler()` promise resolution callback